metaflow/plugins/cards/card_modules/base.html

```diff
@@ -16,9 +16,14 @@
 </head>
 
 <body>
-  <div id="app"></div>
+  <div class="card_app"></div>
   <script>
-    window.__DATA__ = {{{task_data}}}
+    var mfCardDataId = "{{{card_data_id}}}";
+
+    if (!window.__DATA__) {
```

Comment:
Since we're now focusing on this being embedded in other places, should we prefix __DATA?  as that's kind of a standard naming.  so `window.__MF_DATA__` or something?

totally up to you, just a thought.